shared/actions/profile.js

```diff
@@ -1,188 +0,0 @@
-/* @flow */
```

Comment:
we don't have separate profile redux stuff for now. just re-using the tracker code